core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadHandler.java

```diff
@@ -274,6 +285,10 @@ public CompleteMultipartUploadResult call() throws S3Exception {
 
         CreateFilePOptions.Builder optionsBuilder = CreateFilePOptions.newBuilder()
             .setRecursive(true)
+            .setMode(PMode.newBuilder()
+                .setOwnerBits(Bits.ALL)
+                .setGroupBits(Bits.ALL)
+                .setOtherBits(Bits.NONE).build())
```

Comment:
I have no clue how to use these permission bits; I'm just assuming the gRPC does what it implies here and sets the permission bits to `770`.

I also noticed there was no way to set the <owner:group> pair in the CreateFile gRPC. So I assume its implicitly passed via the RPC client context? @yuzhu @apc999 Any insights?